src/.nuget/Microsoft.NETCore.Crossgen/runtime.Linux.Microsoft.NETCore.Crossgen.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CpaotCrossgenRuntimeConfigFile>$(ArtifactsObjDir)crossgen2\crossgen.runtimeconfig.json</CpaotCrossgenRuntimeConfigFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <ArchitectureSpecificToolFile Include="$(BinDir)crossgen2\crossgen" />
+    <ArchitectureSpecificToolFile Include="$(BinDir)crossgen2\crossgen.dll" />
+    <ArchitectureSpecificToolFile Include="$(BinDir)crossgen2\ILCompiler*.dll" />
+    <ArchitectureSpecificToolFile Include="$(BinDir)crossgen2\Microsoft.DiaSymReader.dll" />
+    <ArchitectureSpecificToolFile Include="$(BinDir)crossgen2\System.CommandLine.dll" />
```

Comment:
I see below this is from:

```xml
    <PackageReference Include="System.CommandLine">
      <Version>0.1.0-e160119-1</Version>
    </PackageReference>
```

Which I think refers to this archived corefxlab project: https://github.com/dotnet/corefxlab/tree/master/archived_projects/src/System.CommandLine

Is this known, is a replacement tracked?

(The choice here impacts source-build, we have to be able to build this dependency from source especially since this will distribute it in our output.)

Reply:
It looks removing System.CommandLine is tracked: https://github.com/dotnet/coreclr/issues/26859. Milestone is correct: this needs to happen before 5.0.